api/python/provisioner/ui/README.md

```diff
@@ -0,0 +1,11 @@
+# Provisioner TUI
+
+This is northbound interface TUI for CORTX components that provides terminal UI
+to configure system.
+
+## start with provisioner TUI
+
+1. command to start provisioner TUI 
+```
+python3 api/python/provisioner/ui/main.py
```

Comment:
please consider a one-line change to configure installation of a separate tool for that instead of usage python interpreter (like [here](https://github.com/Seagate/cortx-prvsnr/blob/5b3216c1c39e518c30e3b97608e0064d69530c93/api/python/setup.py#L91))

Reply:
`cortx-setup --node-onboarding`
This would be part of same `provisioner` CLI family, to be renamed to `cortx-setup` or if we want `cortx-prvsnr` 👍